src/slider.jsx

```diff
@@ -199,7 +193,7 @@ const Slider = React.createClass({
   getStyles() {
     const fillGutter = this.getTheme().handleSize / 2;
     const disabledGutter = this.getTheme().trackSize + this.getTheme().handleSizeDisabled / 2;
-    const calcDisabledSpacing = this.props.disabled ? ` -${disabledGutter}px` : '';
+    const calcDisabledSpacing = this.props.disabled ? ` - ${disabledGutter}px` : '';
```

Comment:
It actually matters because the result of:

    this.props.disabled && ` - {disabledGutter}px`
when this.props.disabled is false will be false. So in the string concatenation that happens after:

    width: `calc(${(this.state.percent * 100)}%${calcDisabledSpacing})`
we would have the following string for example: *'calc(75% - false)'*  
I could change the code to behave as expected for that kind of assignment, but I would prefer not, because the advantage of a ternary is that we have the possibility to have a variable with the same type always (like in a functional approach), in that case a *string*. While, in the other case sometimes we have *false* and sometimes a *string*.